dev/io.openliberty.jcache.internal_fat.hazelcast/fat/src/io/openliberty/jcache/internal/fat/JCacheLtpaLoggedOutCookieCacheServerRestartTest.java

```diff
@@ -0,0 +1,185 @@
+/*******************************************************************************
+ * Copyright (c) 2022 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package io.openliberty.jcache.internal.fat;
+
+import static org.junit.Assert.assertTrue;
+
+import java.util.UUID;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import com.ibm.ws.webcontainer.security.test.servlets.FormLoginClient;
+import com.ibm.ws.webcontainer.security.test.servlets.FormLoginClient.LogoutOption;
+import com.ibm.ws.webcontainer.security.test.servlets.SSLFormLoginClient;
+
+import componenttest.annotation.CheckForLeakedPasswords;
+import componenttest.annotation.Server;
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.custom.junit.runner.Mode;
+import componenttest.custom.junit.runner.Mode.TestMode;
+import componenttest.topology.impl.LibertyServer;
+import io.openliberty.jcache.internal.fat.plugins.TestPluginHelper;
+
+/**
+ * Contains distributed JCache logged out server restart cookie cache test for LTPA.
+ * This won't work in hazelcast because hazelcast runs the cache on the Liberty server.
+ */
+@SuppressWarnings("restriction")
+@RunWith(FATRunner.class)
+@Mode(TestMode.FULL)
+public class JCacheLtpaLoggedOutCookieCacheServerRestartTest extends BaseTestCase {
+    @Server("io.openliberty.jcache.internal.fat.ltpa.cookie.cache.restart.1")
+    public static LibertyServer server1;
+
+    @Server("io.openliberty.jcache.internal.fat.ltpa.cookie.cache.restart.2")
+    public static LibertyServer server2;
+
+    private static FormLoginClient formLoginClient1;
+    private static FormLoginClient formLoginClient2;
+
+    @Before
+    public void before() throws Exception {
+        UUID groupName = UUID.randomUUID();
+
+        /*
+         * Start server1.
+         */
+        server1.addInstalledAppForValidation("formlogin");
+        startServer1(server1, groupName.toString(), null, null);
+        waitForCachingProvider(server1, COOKIE_CACHE_NAME);
+        if (TestPluginHelper.getTestPlugin().cacheShouldExistBeforeTest()) {
+            waitForExistingJCache(server1, COOKIE_CACHE_NAME);
+        } else {
+            waitForCreatedJCache(server1, COOKIE_CACHE_NAME);
+        }
+
+        /*
+         * Start server2.
+         */
+        server2.addInstalledAppForValidation("formlogin");
+        startServer2(server2, groupName.toString());
+        waitForCachingProvider(server2, COOKIE_CACHE_NAME);
```

Comment:
This is going in for another PR, but please add this after my PR is merged.

```suggestion
        waitForDefaultHttpsEndpoint(server2);
        waitForCachingProvider(server2, COOKIE_CACHE_NAME);
```